pixel/theme.json

```diff
@@ -150,17 +150,17 @@
             "fontSizes": [
                 {
                     "name": "Small",
-                    "size": "1rem",
+                    "size": ".813rem",
                     "slug": "small"
                 },
                 {
                     "name": "Medium",
-                    "size": "1.5rem",
+                    "size": "clamp(1rem, 0.8611111111111112rem + 0.6944444444444444vw, 1.125rem)",
                     "slug": "medium"
                 },
                 {
                     "name": "Large",
-                    "size": "1.75rem",
+                    "size": "1.5rem",
```

Comment:
when fluid is enabled globally, and explicit `fluid: false` is required here to apply fixed size.